src/main/kotlin/lotto/model/Coincidence.kt

```diff
@@ -1,8 +1,9 @@
 package lotto.model
 
-enum class Coincidence(val coincidenceCount: Int, val prizeMoney: Int) {
-    THREE(3, 5_000),
-    FOUR(4, 50_000),
-    FIVE(5, 1_500_000),
-    SIX(6, 2_000_000_000);
+enum class Coincidence(val coincidenceCount: Int, val prizeMoney: Int, val hasBonusNum: Boolean) {
+    THREE(3, 5_000, false),
+    FOUR(4, 50_000, false),
+    FIVE(5, 1_500_000, false),
+    FIVE_WITH_BONUS(5, 30_000_000, true),
+    SIX(6, 2_000_000_000, false);
```

Comment:
로또의 순위는 1 등부터 5 등까지입니다. 아래의 글이 도움 되면 좋겠어요.
https://dhlottery.co.kr/gameResult.do?method=myWin

```suggestion
    FIFTH(3, 5_000, false),
    FORTH(4, 50_000, false),
    THIRD(5, 1_500_000, false),
    SECOND(5, 30_000_000, true),
    FIRST(6, 2_000_000_000, false);
```